### ISSUE TITLE
Build bundle with --debug

### ISSUE THREAD

**Issue by lauff** [jshimko/meteor-launchpad#90]
Hi & Happy New Year!

Is it possible to build a bundle with the --debug parameter?

Thanks
Markus

**Comment by jshimko** [comment]
Do you mean inside the container [here](https://github.com/jshimko/meteor-launchpad/blob/master/scripts/build-meteor.sh#L27)?

**Comment by lauff** [comment]
Yes, it would be nice if it would be possible to add further parameters (such as "--debug"). 
I guess the easiest would be to add an environment variable "METEOR_BUILD_PARAMS".

meteor build --directory $APP_BUNDLE_DIR --server-only $METEOR_BUILD_PARAMS

e.g. METEOR_BUILD_PARAMS="--debug"

Thanks for your help
Markus
